### ISSUE TITLE
Crash 

### ISSUE THREAD

**Issue by Mridul56** [layerhq/Atlas-iOS#1388]
Layer is crashing ,  due to exception , which xcode is unable to show the trace. 
Please check the trace in attached screenshot

Version : 1.0.31
<img width="479" alt="screen shot 2017-05-14 at 2 10 34 pm" src="https://cloud.githubusercontent.com/assets/11690951/26032361/45a478b8-38af-11e7-9a39-de104f7208a0.png">

**Comment by magneticrob** [comment]
I've spoken to Layer about this directly over email and on a call, they're aware of it and working on a new version which should fix this. It's a server side issue so there are updates to the server as well as LayerKit to stop it from crashing when it experiences this. 

If you send an email to support@layer.com you might have more luck.

**Comment by Mridul56** [comment]
@magneticrob  Thanks for replying waiting for Layer to fix this issue, Is there any possibility of handling this issue at Atlas side  or at application side to avoid that crash till the time new update comes ?

**Comment by magneticrob** [comment]
Kind of. On my side for development purposes here's how I fixed it temporarily:

- Downgrade to an older version of LayerKit (I used v0.25.2)
- Create a new Layer user (bit annoying this, but your Layer account is a bit screwed at this point until the new version of LayerKit comes out)

We managed to avoid this hitting production luckily - our app in the store is running an older version of LayerKit. Try sending an email off to support@layer.com if you haven't and they should be able to help you out.
